packages/e2e-tests/test/e2e-fle.spec.ts

```diff
@@ -716,11 +716,158 @@ describe('FLE tests', function () {
       expect(parseInt(dekCount.trim(), 10)).to.equal(1);
     });
 
-    context('using rangePreview algorithm', function () {
-      // TODO(MONGOSH-1742): Server 8.0 drops "rangePreview" algorithm and adds
-      // "range". Re-enable these when the change is finalized
-      skipIfServerVersion(testServer, '>= 8.0.0-alpha');
+    context(
+      'using rangePreview algorithm when server is > 7 and < 8',
+      function () {
+        skipIfServerVersion(testServer, '>= 8.0.0');
```

Comment:
Could we use the same test code for 7.x and 8.x+ and only switch between `range` and `rangePreview` according to the server version?

Reply:
yeah good point, will do it!